android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
     android:icon="@drawable/ic_launcher"
     android:label="@string/app_name"
     android:roundIcon="@drawable/ic_launcher_round"
+    android:usesCleartextTraffic="true"
```

Comment:
This should only be true for debug builds? Why did you need to add it now?

Reply:
The embedded browser was blocking regular http:// traffic with the error
```
net::ERR_CLEARTEXT_NOT_PERMITTED
```